ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/compact/provision/cucushift-installer-rehearse-aws-ipi-compact-provision-chain.yaml

```diff
@@ -0,0 +1,18 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-compact-provision
```

Comment:
I guess do not need to introduce provision/deprovision chain, we can similar things like https://github.com/openshift/release/pull/36016

Reply:
OK, I will use a existing chain.Thank you.